web/urls.py

```diff
@@ -2,6 +2,8 @@
 
 from . import views
 
+app_name = 'web'
```

Comment:
Ok, I see you added this (therefore making the need to add "web" to assorted other areas. I'm curious why you added it as I was kind of thinking it wasn't needed for a single-app project. Is it more standard practice or am I breaking things by not having it?

Reply:
It just a standard practice.